src/index.html

```diff
@@ -2,15 +2,57 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+ 
+    <link href="fonts/Avenir-Book.ttf" rel="stylesheet">
+    <form 
+          action="#" 
+          method="post" 
+          data-qa="big"
+    >
```

Comment:
```suggestion
    <form 
      action="#" 
      method="post" 
      data-qa="big"
    >
```

check cody style for all cases
checklist (point 5) https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md